juntagrico/tests/test_jobs.py

```diff
@@ -192,6 +192,34 @@ def testAssignmentEditByCoordinator(self):
         self.testAssignmentEdit(self.area_admin)
         self.testAssignmentDelete(self.area_admin)
 
+    def testAdminNotificationContainsMemberInfo(self):
+        """
+        Regression test: Admin notification emails for assignment changes/removals
+        should contain both the editor's and the member's name and email address.
+        Previously, the member variable was not passed to the template, causing
+        the member's name and email to be missing from the notification.
+        """
+        # Admin removes a member's assignment
+        self.assertPost(reverse('assignment-edit', args=[self.job2.pk, self.member.pk]),
+                        {'edit-slots': 0}, 302, self.admin)
+
+        # Check that 2 emails were sent: one to member, one to coordinator
+        self.assertEqual(len(mail.outbox), 2)
+
+        # Get the admin notification email (sent to coordinator)
+        admin_email = mail.outbox[1]
+        self.assertEqual(admin_email.recipients(), ['email_contact@example.org'])
+
+        # Verify the email body contains both editor and member information
+        email_body = admin_email.body
+
+        # Check for editor (admin) information
+        self.assertIn('admin@email.org', email_body,
+                      "Admin notification should contain editor's email address")
+
+        # Check for member information - this was the bug: member email was missing
+        self.assertIn('email1@email.org', email_body,
```

Comment:
Just add this check to `testAssignmentEdit` (see previous comment) and drop the rest.